modelcontextprotocol/server.py

```diff
@@ -424,6 +428,46 @@ def update_assets_tool(
             ]
         )
 
+        # Update readme for a single asset with Markdown
+        result = update_assets_tool(
+            assets={
+                "guid": "asset-guid-here",
+                "name": "Asset Name",
+                "type_name": "Asset Type Name",
+                "qualified_name": "Asset Qualified Name"
+            },
+            attribute_name="readme",
+            attribute_values=['''# Customer Data Table
+            Contains customer transaction records for analytics.
+            **Key Info:**
+            - Updated daily at 2 AM
+            - Contains PII data
+            - [Documentation](https://docs.example.com)''']
+        )
+
+        # Update multiple assets with different user descriptions
```

Comment:
Do we need this?

Reply:
Resolved via : [5bad274](https://github.com/atlanhq/agent-toolkit/pull/63/commits/5bad274c8d3fbef2b0ca18c3ad0813c3812b988a)